### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by lauraguerra1**
Description
Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,35 +11,61 @@ Javascript | HTML | CSS | REST API
 </div>
 
 ## Abstract: 
-This is a recipe website that uses the fetch API to display information about a variety of recipes. This web app also allows searching recipes by name, ingredients, tags, and allows users to save their favorite recipes! This is a collaborative project, created to ensure cohesive usage of newly learned concepts such as Rest APIs and array prototype methods.
+This is a recipe website that uses the fetch API to display information about a variety of recipes. This web app also allows searching recipes by name, ingredients, tags, and allows users to save their favorite recipes! This is a collaborative project, created to ensure cohesive usage of newly learned concepts such as Rest APIs, and array prototype methods to implement CRUD functionality. 
 ### User Stories: 
 - Users can click on a recipe to view more information.
 - Users are able to save recipes from the main landing page as well as from inside the recipe display.
```
Comment (taranveersingh93) [inline README.md]: save and delete?

**Review by taranveersingh93 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lauraguerra1) [reply README.md]: ohh good point, done now

**Review by mbenfowler — APPROVED** [review]
looks great!

**Review by taranveersingh93 — APPROVED** [review]

**Comment by mbenfowler** [comment]
[update README](https://trello.com/c/dBVWNVpg/60-update-readme)

**Pull request merged**